docs/Configuration.md

```diff
@@ -822,73 +822,62 @@ When using multi-project runner, it's recommended to add a `displayName` for eac
 
 Default: `undefined`
 
-Use this configuration option to add custom reporters to Jest. A custom reporter is a class that implements `onRunStart`, `onTestStart`, `onTestResult`, `onRunComplete` methods that will be called when any of those events occurs.
-
-If custom reporters are specified, the default Jest reporters will be overridden. To keep default reporters, `default` can be passed as a module name.
-
-This will override default reporters:
+Use this configuration option to add reporters to Jest. Optionally, a reporter can be configured by passing options object as a second argument of a tuple:
 
 ```json
 {
-  "reporters": ["<rootDir>/my-custom-reporter.js"]
+  "reporters": [
+    "default",
+    ["<rootDir>/custom-reporter.js", {"banana": "yes", "pineapple": "no"}]
+  ]
 }
 ```
 
-This will use custom reporter in addition to default reporters that Jest provides:
+:::tip
 
-```json
-{
-  "reporters": ["default", "<rootDir>/my-custom-reporter.js"]
-}
-```
+Take a look at a list of [awesome reporters](https://github.com/jest-community/awesome-jest#reporters) from Awesome Jest.
 
-Additionally, custom reporters can be configured by passing an `options` object as a second argument:
+If custom reporters are specified, the default Jest reporters will be overridden. If you wish to keep them, `'default'` can be passed as a module name:
 
 ```json
 {
   "reporters": [
     "default",
-    ["<rootDir>/my-custom-reporter.js", {"banana": "yes", "pineapple": "no"}]
+    ["jest-junit", {"outputName": "junit-report.xml", "suiteName": "some-name"}]
   ]
 }
 ```
 
-Custom reporter modules must define a class that takes a `GlobalConfig` and reporter options as constructor arguments:
+:::
 
-Example reporter:
+Custom reporter module must export a class that takes `globalConfig`, `reporterConfig` and `reporterContext` as constructor arguments and implements at least `onRunComplete()` method (for the full list of methods and argument types see `Reporter` interface in [packages/jest-reporters/src/types.ts](https://github.com/facebook/jest/blob/main/packages/jest-reporters/src/types.ts)):
```

Comment:
What's the difference between `reporterConfig` and `reporterContext`? Can they be combined?

Reply:
Ah, snap below sorta explained